### PR TITLE
expose epsilon parameter to allow precision to to be specified

### PR DESCRIPTION
**Pull request by ChuckHastings**
Per user request, this PR exposes the epsilon value that controls the precision where the Hungarian algorithm determines that a value has been reduced to sufficiently close to 0.

### CODE REVIEW

**Review by cjnolet — APPROVED** [review]
LGTM

File: cpp/include/raft/lap/lap_kernels.cuh [hunk 1]
```diff
@@ -48,20 +48,18 @@ const int AUGMENT{4};
...
+bool __device__ near_zero(weight_t w, weight_t epsilon) {
+  return ((w > -epsilon) && (w < epsilon));
 }
 
 template <>
-bool __device__ near_zero<float>(float w) {
-  const float epsilon{1e-6};
-  return ((w > -epsilon) && (w < epsilon));
+bool __device__ near_zero<int32_t>(int32_t w, int32_t epsilon) {
+  return (w == 0);
```
Comment (seunghwak) [inline cpp/include/raft/lap/lap_kernels.cuh:57]: Why aren't we using epsilon here?

**Review by seunghwak — APPROVED** [review]
I have one question, but besides the question, looks good to me.

File: cpp/include/raft/lap/lap_kernels.cuh (hunk 1 above)
Comment (ChuckHastings) [reply cpp/include/raft/lap/lap_kernels.cuh:57]: For integer weights we can do an exact comparison to zero.  For floating point values we need to do an approximate comparison.

**Comment by cjnolet** [comment]
@gpucibot merge

**Pull request merged**